cfgov/jinja2/owning-a-home/explore-rates/index.html

```diff
@@ -28,568 +28,565 @@
 {% endblock %}
 
 {% block body_content %}
-    <div class="content_wrapper">
-        <div class="content_main u-pb0">
```

Comment:
The [viz checker](https://github.com/cfpb/cfgov-visual-regression) is showing that removing `content_main` is causing paragraphs to not get wrapped on the rate checker:

| before | after |
|--------|-------|
| <img width="1210" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/1060248/8e573010-589d-4672-9af1-db8be53f5e2d"> | <img width="1210" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/1060248/9c380ed6-ed59-48d2-94b5-8dcad65ea073"> |